src/gui/plugins/component_inspector/ComponentInspector.cc

```diff
@@ -870,13 +936,6 @@ void ComponentInspector::Update(const UpdateInfo &,
       if (comp)
         setData(item, comp->Data());
     }
-    else if (typeId == components::WorldPoseCmd::typeId)
```

Comment:
Was this removed on purpose?

Reply:
Yes, this was removed on purpose. The `WorldPoseCmd`, as far as I could tell, is a component used by the transform control tool to inform the physics system that the user has transformed model. I don't think it has value in the component inspector.

Reply:
I've looked at it in the past to check that my command was properly propagated

Reply:
I'd prefer to not have a such a low-level detail displayed upfront on the GUI because it's confusing and appears only when the transform tool is used.

Reply:
I think there is a disconnect between my goals and your goals which could be resolved by keeping the ComponentInspector as a tool that strictly displays components for the purpose of debugging simulation and creating a separate plugin that is more a ParameterEditor that is a user-friendly way to configure simulation while hiding some underlying details.